src/packages/ganache/README.md

```diff
@@ -476,7 +476,55 @@ Server:
   -p, --server.port, --port             Port to listen on.
                                         deprecated aliases: --port
                                                                                       [number] [default: 8545]
+```
+
+### Detached Instances
+
+Ganache can be started as a background instance via the CLI by providing the following argument (along with any combination
+of the Ganache startup arguments above):
+
+```
```

Comment:
So I had a look around, and take this with a grain of salt, because I don't have anything authoritative,  but:

https://github.com/github/linguist/blob/3c3b037910006fc2f1a9bb34b2c4e9cde062206c/lib/linguist/languages.yml shows the languages that github knows, and the various aliases.

- `bash` maps to `Shell`
- `console` maps to `ShellSession`

I believe that the difference is that a `Shell` block should be a shell _script_, where as a `ShellSession` should be a series of commands and responses from a session.

Reply:
So in these examples, ` ```console` is correct